config/docker-compose.yml.tim

```diff
@@ -0,0 +1,32 @@
+version: '2'
```

Comment:
The latest version is `v3` . But `volumes_from` with `v3` is not intuitive 😢 . So I use `v2.
https://stackoverflow.com/questions/42232051/docker-compose-volumes-from-equivalent-with-version-3